docs/user/firststeps.rst

```diff
@@ -520,30 +530,42 @@ and other commands.
 
 .. note::
 
-    Did you get an error like this?
+    Did you accidentally run the python command first? You would get
+    an error like this
+
+    .. code-block:: console
+
+        # Error: The repository is dirty. Please use the "git" command to clean it.
+        # On branch master
+        # Your branch is up to date with 'origin/master'.
+        # Untracked files:
+        # (use "git add <file>..." to include in what will be committed)
+        #         data/output/
+
+    Remove the untracked files and this time execute ``only`` the renku command
+
+    .. code-block:: console
+
+        rm data/output/*
+        renku run python src/00-FilterFlights.py data/201901_us_flights_1/2019-01-flights.csv.zip data/output/2019-01-flights-filtered.csv
+
+.. note::
+
+    Did you get an error like this instead?
```

Comment:
Note - this should not happen with `renku>=0.9.0` - if it does, we need to reopen the appropriate issue

Reply:
I was using renku 0.9.0 and this happened again :thinking: 

https://dev.renku.ch/projects/lorenzo.cavazzi.tech/flights-tutorial/

Reply:
Oh wait, you are right, I tried to reproduce this and it's not happening anymore. I am wondering how it comes that this happened the first time I run through this

Reply:
ah wait, actually this change doesn't make sense - the error should happen when the data is not pulled yet but in that case you won't get a "file not found" error, you'll get the "bad zip archive" error instead. No?

Reply:
that is the error below, this one happens when accidentally executing also the first part of the code that is the "non-renku" way

Reply:
oh I probably read it out of order 🤦‍♂